doc/classes/EditorSettings.xml

```diff
@@ -157,430 +157,691 @@
+			If [code]true[/code], the mouse's additional side buttons will be usable to navigate in the script editor's file history. Set this to [code]false[/code] if you're using the side buttons for other purposes (such as a push-to-talk button in a VoIP program).
 		</member>
 		<member name="text_editor/navigation/show_minimap" type="bool" setter="" getter="">
+			If [code]true[/code], draws an overview of the script near the scroll bar. The minimap can be left-clicked to scroll directly to a location in an "absolute" manner.
 		</member>
 		<member name="text_editor/navigation/smooth_scrolling" type="bool" setter="" getter="">
+			If [code]true[/code], allows scrolling in sub-line intervals and enables a smooth scrolling animation when using the mouse wheel to scroll.
+			[b]Note:[/b] [member text_editor/navigation/smooth_scrolling] currently behaves poorly in projects where [member ProjectSettings.physics/common/physics_fps] has been increased significantly from its default value ([code]60[/code]). In this case, it is recommended to disable this setting.
 		</member>
 		<member name="text_editor/navigation/stay_in_script_editor_on_node_selected" type="bool" setter="" getter="">
+			If [code]true[/code], prevents automatically switching between the Script and 2D/3D screens when selecting a node in the Scene tree dock.
 		</member>
 		<member name="text_editor/navigation/v_scroll_speed" type="int" setter="" getter="">
+			The number of pixels to scroll with every mouse wheel increment. Higher values make the script scroll by faster when using the mouse wheel.
+			[b]Note:[/b] You can hold down [kbd]Alt[/kbd] while using the mouse wheel to temporarily scroll 5 times faster.
 		</member>
 		<member name="text_editor/script_list/show_members_overview" type="bool" setter="" getter="">
+			If [code]true[/code], displays an overview of the current script's member variables and functions at the left of the script editor. See also [member text_editor/tools/sort_members_outline_alphabetically].
 		</member>
 		<member name="text_editor/theme/color_theme" type="String" setter="" getter="">
+			The syntax theme to use in the script editor.
+			You can save your own syntax theme from your current settings by using [b]File &gt; Theme &gt; Save As...[/b] at the top of the script editor. The syntax theme will then be available locally in the list of color themes.
+			You can find additional syntax themes to install in the [url=https://github.com/godotengine/godot-syntax-themes]godot-syntax-themes[/url] repository.
 		</member>
 		<member name="text_editor/theme/line_spacing" type="int" setter="" getter="">
+			The space to add between lines (in pixels). Greater line spacing can help improve readability at the cost of displaying fewer lines on screen.
 		</member>
 		<member name="text_editor/tools/create_signal_callbacks" type="bool" setter="" getter="">
 		</member>
```

Comment:
This setting is unused, so I made another PR to remove it: https://github.com/godotengine/godot/pull/64892